smtk-import/smartrak.c

```diff
@@ -418,6 +418,95 @@ static void smtk_build_tank_info(MdbHandle *mdb, struct dive *dive, int tanknum,
 }
 
 /*
+ * Under some circustances we can get the same tank from DC and from
+ * the smartrak DB. Will use this utility to check and clean .
+ */
+bool is_same_cylinder(cylinder_t *cyl_a, cylinder_t *cyl_b)
+{
+	// different gasmixes (non zero)
+	if (gasmix_distance(&cyl_a->gasmix, &cyl_b->gasmix) != 0 &&
+	    cyl_a->gasmix.o2.permille != 0 &&
+	    cyl_b->gasmix.o2.permille != 0)
```

Comment:
Sorry, sent a PR with a patch.  Don't even understand how this ever worked.  gasmix_distance() has given too many problems, so I finally dropped it.